performance/include/traccc/performance/details/projector.hpp

```diff
@@ -47,12 +47,4 @@ struct projector<traccc::bound_track_parameters<algebra_t>> {
     }
 };
 
-template <>
-struct projector<fitting_result<traccc::default_algebra>> {
-    static constexpr bool exists = true;
-
-    float operator()(const fitting_result<traccc::default_algebra>& i) {
-        return static_cast<float>(i.trk_quality.ndf);
-    }
-};
```

Comment:
Where did this go?

Reply:
The `traccc::fitting_result` object is no more.

And this class is fundamentally not needed with the SoA data. We don't need to do anything to "project" the `ndf` variable into a vector. We can just access the array holding the `ndf` variable "natively", without any transformation.